core/jazz_ui/Jenkinsfile_Platform

```diff
@@ -144,12 +147,12 @@ def loadBuildModules(buildModuleUrl){
 				[credentialsId: repo_credential_id, url: buildModuleUrl]
 			]])
 
-      def result_json_string = readFile("jazz-installer-vars.json")
-      config_module = load "config-loader.groovy"
-      config_loader = config_module.initialize(result_json_string)
+      configModule = load "config-loader.groovy"
+		  configLoader = configModule.loadConfigData(aws_credential_id, region, instance_prefix)
```

Comment:
indentation

Reply:
done. It is seeing correct in my local and in remote branch, not sure, why this seeing in this PR